test/kubernetes_unittest.cc

```diff
@@ -1215,4 +1184,705 @@ TEST_F(KubernetesTestFakeServer, MetadataQuery) {
   EXPECT_EQ(pod_metadata->ToString(), m[3].metadata().metadata->ToString());
 }
 
+class KubernetesTestFakeServerOneWatchRetryNodeLevelMetadata
+  : public KubernetesTest {
```

Comment:
If you inherit from `KubernetesTestFakeServer`, you won't need to repeat `SetUp` and add the `server` data field...

Reply:
Done.